sdk/examples/xo_python/sawtooth_xo/xo_client.py

```diff
@@ -126,7 +127,8 @@ def _send_xo_txn(self, name, action, space=""):
             dependencies=[],
             payload_encoding="csv-utf8",
             payload_sha512=_sha512(payload),
-            batcher_pubkey=self._public_key
+            batcher_pubkey=self._public_key,
+            nonce=time.time().hex().encode()
```

Comment:
time.time().hex().encode() is what we have for intkey, so I'm sure it works. I can't remember if this is a python2/python3 difference with protobuf, or the qualities of a nonce only matter that you make it somewhat random.